.github/workflows/config/spelling_allowlist.txt

```diff
@@ -233,9 +245,12 @@ fmt
 fourier
 frontend
 frontends
+github
+gpus
 hadamard
 hamiltonian
 homogenous
+https
```

Comment:
This is not a word. Why are we checking the spelling of a URL? That doesn't make much sense.